### ISSUE TITLE
Feature request – add an argument batchsnatched

### ISSUE THREAD

**Issue by damonjavert** [damonjavert/jps2sm#27]
2021-05-07T10:16:20Z

Currently --batchuploaded and --batchseeding work by going to different URLs at JPS, and they are sorted by Name as otherwise --batchseeding does not work as it sorts by last announce which is never helpful - it means the order is effectively random.

Uploaded and seeding is determined by the `mode` var, you can see this on line 106 in jps2sm.py:

```
        useruploadurl = fr"https://jpopsuki.eu/torrents.php?page={i}&order_by=s1&order_way=ASC&type={mode}&userid={user}&disablegrouping=1"
```

`snatched` will work for this in the URL for our case.

Sorting by Name is achieved by `&order_by=s1` on the same line, we only sort by Name in order to allow --batchseeding to work correctly, as described above. For this issue though we can sort by snatched time for --batchsnatched. This is achieved by `&order_by=s3` , the same for --batchuploaded, the s3 field is 'Added' on this page.

In summary:

* Add --batchsnatched via `type=snatched` in the URL via using the `mode` var.
* For --batchsnatched and --batchuploaded use sorting via 'Snatched' and 'Added' respectively, both can use `&order_by=s3` for this.
* For --batchseeded this should remain as 'Name' as we do not get an 'Added' field in the output, we should continue to use `&order_by=s1`

Bonus points:

Not required for this PR, but it would be cool if we were to add an --order argument so the user could upload their most recently snatched/uploaded torrents, and also order by Name if they wish.

**Comment by damonjavert** [comment]
65b69efe7f164fcdf666d8bbf38bbf631c01fba4
